### PR TITLE
[FR] Translation introduction

### PR DESCRIPTION
**Pull request by Gastonia02**
fr_introduction_page-0001

French translation of the introduction section

### CODE REVIEW

File: translations/introduction.tex/fr.po [hunk 1]
```diff
@@ -79,3 +106,6 @@ msgid ""
 "  \\centering\n"
 "  \\includegraphics[width=\\linewidth, height=\\myspace, keepaspectratio]{\\art/hydra.png}"
 msgstr ""
+"[blanker]\n"
+"  \\centering\n"
+"  \\includegraphics[width=\\linewidth, height=\\myspace, keepaspectratio]{\\art/hydra.png}"
```
Comment (qwrtln) [inline translations/introduction.tex/fr.po:111]:
```suggestion
```
If there's nothing to translate, you can skip it. `po4a` picks up the original in this case.

File: translations/introduction.tex/fr.po [hunk 2]
```diff
@@ -70,6 +76,27 @@ msgid ""
 "}\n"
 "\\vfill"
 msgstr ""
+"{2}\n"
+"\\textbf{Notice}: Dans ce livret, les termes et éléments de jeu sont écrits avec leur première lettre en Majuscule.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/fr.po:80]: Is this how you say "Notice" in French?

File: translations/introduction.tex/fr.po [hunk 3]
```diff
@@ -70,6 +76,27 @@ msgid ""
...
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/spellpower.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/movement.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"}\n"
+"\\vfill\n"
+"\\columnbreak\n"
+"\\note{10}{Les exceptions et les notes avec des \\hyperlink{Heroes of Might and Magic III}{Liens Hypertextes ambre} sont expliquées dans des cases comme celle-ci.\n"
+"  \\medskip\\\\\n"
+"  Les conflits de règles suivent cet ordre de priorité : Carte du Joueur, Carte d’Unité, Plateau de Ville, Livret de Mission, Livret de Base.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/fr.po:97]:
```suggestion
"  Les conflits de règles suivent cet ordre de priorité: Carte du Joueur, Carte d’Unité, Plateau de Ville, Livret de Mission, Livret de Base.\n"
```

File: translations/introduction.tex/fr.po [hunk 4]
```diff
@@ -43,6 +46,9 @@ msgid ""
 "Magic III: The Board Game}} is a tactical strategy RPG board game for 1-3 "
 "players using the core box set, or more when using expansions."
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: The Board Game}} est un jeu de stratégie pour 1 à 3 "
+"joueurs utilisant le set de base."
```
Comment (qwrtln) [inline translations/introduction.tex/fr.po:51]: I think the info about expansions is missing? If you're just copying from @Legrecofr's translation, it's not up-to-date.

File: translations/introduction.tex/fr.po (hunk 2 above)
Comment (Gastonia02) [reply translations/introduction.tex/fr.po]: Changed it

File: translations/introduction.tex/fr.po (hunk 3 above)
Comment (Gastonia02) [reply translations/introduction.tex/fr.po:97]: In french, a space is required before and after the ":"

Comment (qwrtln) [reply translations/introduction.tex/fr.po:97]: Didn't know that, thanks!

**Comment by qwrtln** [comment]
Please translate "Introduction", line 28.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**